wandb/jupyter.py

```diff
@@ -359,7 +359,7 @@ def save_history(self):
                 },
             )
             state_path = os.path.join("code", "_session_history.ipynb")
-            wandb.run._config_callback(
+            wandb.run._set_config_wandb(
                 val=state_path, key=("_wandb", "session_history")
```

Comment:
I think the common case is just setting a single key inside _wandb so the interface should probably be:
key: str, val: Any

hide the nested key stuff from the caller.

```
wandb.run._set_config_wandb("session_history", state_path)
```

And you have a mypy lint error

Reply:
Ah yeah, you're right.